src/argilla/client/feedback/metrics/agreement_metrics.py

```diff
@@ -36,7 +36,9 @@
 
 
 def prepare_dataset_for_annotation_task(
-    dataset: Union["FeedbackDataset", "RemoteFeedbackDataset"], question_name: str
+    dataset: Union["FeedbackDataset", "RemoteFeedbackDataset"],
+    question_name: str,
+    response_status: Optional[Union[ResponseStatusFilter, str]] = ResponseStatusFilter.submitted.value,
```

Comment:
I would call this `filter_by` specifically. Also, can you align this with https://docs.argilla.io/en/latest/practical_guides/fine_tune.html#filter-and-sort-datasets-for-training?

Reply:
Sure. But in this case should we let the default to submitted, or also without any filter?

Reply:
I am unsure about this. Perhaps it is cleanest to default to None because we do that in other places too? Also, I think we should add `sort_by` and `max_records`.

Reply:
Done